packages/ui/src/components/va-alert/VaAlert.vue

```diff
@@ -2,6 +2,7 @@
   <transition
     v-if="valueComputed"
     name="fade"
+    ref="alert"
```

Comment:
Why do we need this?

Reply:
It's a a rudiment from debugging, I'll remove it.